src/common/ConfUtils.cc

```diff
@@ -102,21 +102,24 @@ parse_file(const std::string &fname, std::deque<std::string> *errors,
   FILE *fp = fopen(fname.c_str(), "r");
   if (!fp) {
     ret = -errno;
+    ostringstream oss;
+    oss << "parse_file: failed to fopen '" << fname << "': " << cpp_strerror(ret) <<".";
```

Comment:
On 24-6-2016 03:48, Kefu Chai wrote:
> In src/common/ConfUtils.cc
> <https://github.com/ceph/ceph/pull/9862#discussion_r68343747>:
> 
>> @@ -102,21 +102,24 @@ parse_file(const std::string &fname, std::deque<std::string> *errors,
>>    FILE *fp = fopen(fname.c_str(), "r");
>>    if (!fp) {
>>      ret = -errno;
>> +    ostringstream oss;
>> +    oss << "parse_file: failed to fopen '" << fname << "': " << cpp_strerror(ret) <<".";
> 
> could use |__func__| instead of hardcode it to "parse_file".
> 

Or even better |__func__ << ":" << __LINE__| which would help a lot
whilst debuging testing.